src/test/java/com/ea/restaurant/service/impl/UserServiceImplTest.java

```diff
@@ -0,0 +1,87 @@
+package com.ea.restaurant.service.impl;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.mockito.Mockito.*;
```

Comment:
Remove these static imports, because it could be a little bit confusing when having other classes with similar methods

Reply:
Done.